index.js

```diff
@@ -13,9 +13,13 @@
 
 const fs = require("fs");
 const path = require("path");
-const { Transform } = require("assemblyscript/cli/transform");
-const assemblyscript = require("assemblyscript");
-const { Node, NodeKind, AssertionKind, ASTBuilder } = assemblyscript;
+const {
+  Node,
+  NodeKind,
+  AssertionKind,
+  ASTBuilder,
+  Transform,
+} = require("visitor-as/as");
```

Comment:
@willemneal I'm just wondering why you add extra `visitor-as/as` dependency which just reexport standard classes? =)

Reply:
See above. It's too ensure you don't import a different assemblyscript rather it finds it in node's module cache. Trust me I ran into this issue too many times.

Reply:
@MaxGraey this is actually super useful, because I ran into this problem during local dev/testing, that my project’s `require('assemblyscript')` loaded a _different_ module to what `as-inliner` was loading and then things throw because global `const`s are being overriden.

Reply:
I see. Is it node.js / resolver issue?

Reply:
Yeah it's because it resolves from the transform's perspective, which could be different than the original `asc` that called the transform.